include/openspace/scene/scene.h

```diff
@@ -233,6 +235,26 @@ class Scene : public properties::PropertyOwner {
      */
     static scripting::LuaLibrary luaLibrary();
 
+    /**
+     * Sets a property using the 'properties' contents of a profile. The function will
+     * loop through each setProperty command. A property may be set to a bool, float,
+     * or string value (which must be converted because a Profile stores all values
+     * as strings)
+     *
+     * \param p The Profile to be read.
+     */
+    void setFromProfile_properties(const Profile& p);
+
+    /**
+     * Accepts string version of a property value from a profile, converts it to the
+     * appropriate type, and then pushes the value onto the lua state.
+     *
+     * \param L the lua state to push value to
+     * \param value string representation of the value with which to set property
+     */
+    void property_pushValueFromProfileToLuaState(ghoul::lua::LuaState& L,
```

Comment:
This function doesn't access anything from the Scene class and just be defined as a local function in the cpp file instead

Reply:
Changed to independent function.